packages/core/src/App/Components/Elements/Notifications/__tests__/empty-notification.spec.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { EmptyNotification } from '../empty-notification';
+import { render, screen } from '@testing-library/react';
+
+describe('EmptyNotification Component', () => {
+    it('should render EmptyNotification component', () => {
+        render(<EmptyNotification />);
+        const text = screen.getByText(/No notifications/i);
```

Comment:
just to remind: there is no needs to use capital letter while using /i in regexp

Reply:
Good point thank u @yauheni-kryzhyk-deriv :pray:

Reply:
Will change this